pkg/controller/cloud/nodecontroller.go

```diff
@@ -55,6 +65,13 @@ const (
 
 	// The amount of time the nodecontroller should sleep between retrying NodeStatus updates
 	retrySleepTime = 20 * time.Millisecond
+
+	//Taint denoting that a node needs to be processed by external cloudprovider
+	CloudTaintKey = "ExternalCloudProvider"
+
+	nodeStatusUpdateFrequency = 10 * time.Second
```

Comment:
I've been thinking on and off about making this configurable. That might be the best solution for this. The default in the KCM is 10 * time.Second. So, I'll leave the default at that